### PR TITLE
Use node-gyp to build crypto-cpp

### PR DESCRIPTION
**Pull request by utay**
- Remove src/crypto-cpp and cmake in eed2ff6498b9de4fd46b7ada1bcd03953763c0ef 
- In 4be787a
  - Add starkware-libs/crypto-cpp as a submodule 
  - Import src/starkware/crypto.js 
  - Use binding.gyp to build the lib on `yarn install`
  - Use package.json's `files` to list packaged files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,13 +4,15 @@
   "author": "Sorare",
   "license": "MIT",
   "version": "1.1.0",
```
Comment (kaderate) [inline package.json]: You should bump minor version.

File: binding.gyp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        "deps/crypto-cpp/src/starkware/crypto/ffi/utils.cc",
+        "deps/crypto-cpp/src/starkware/crypto/ffi/ecdsa.cc",
+        "deps/crypto-cpp/src/starkware/crypto/ecdsa.cc",
+        "deps/crypto-cpp/src/starkware/crypto/elliptic_curve_constants.cc",
+        "deps/crypto-cpp/src/starkware/starkex/order.cc",
+        "deps/crypto-cpp/src/starkware/algebra/prime_field_element.cc"
+      ],
+      "include_dirs": ["deps/crypto-cpp/src"],
+      "cflags_cc": ["-std=c++17", "-Wall", "-Wextra", "-fno-strict-aliasing", "-fPIC", "-fexceptions", "-O3"]
+    }
```
Comment (kaderate) [inline binding.gyp:28]: Should I add the MacOS specific configuration?

File: package.json (hunk 1 above)
Comment (utay) [reply package.json]: Done

File: binding.gyp (hunk 2 above)
Comment (utay) [reply binding.gyp:28]: Yes please!

Comment (utay) [reply binding.gyp:28]: LGTM

**Pull request merged**